insights/parsers/ip.py

```diff
@@ -577,3 +614,92 @@ class Ipv6Neigh(IpNeighParser):
     Class to parse ``ip -6 neigh show nud all`` command output.
     """
     pass
+
+
+@parser("ip-s_link")
```

Comment:
You need to add the new spec to the new `insights/specs.py` file, this is the new spec file for the master branch. And for now, we'd better keep the `insights/specs.py` and `insights/config/specs.py` synchronized.

And you need to rename the spec to change the `-` to `_`, since the `-` and `.` are not permitted for a variable name and let the old name `ip-s_link` be the alias of the new name, for details please refer to the content of `insights/specs.py`, there are many examples.

Reply:
I mean you can rename the new spec to 'ip_s_link'.

Reply:
And use it like here without the quotation marks:
~~~
@parser(ip_addr)
...
@parser(ip_s_link)
~~~